generated/nidmm/session.py

```diff
@@ -1943,7 +1943,7 @@ def _init_with_options(self, resource_name, id_query=False, reset_device=False,
                 | VI_FALSE          | 0 | Don't Reset  |
                 +-------------------+---+--------------+
 
-            option_string (str): | Sets the initial value of certain attributes for the session. The
+            option_string (dict or str): | Sets the initial value of certain attributes for the session. The
```

Comment:
python_name should not be option_string anymore...
options?
option_dict?

Reply:
Renamed to options